apidoc/Titanium/Android/Activity.yml

```diff
@@ -294,6 +294,14 @@ events:
         type: Titanium.Android.Intent
     since: '3.0.0'
 
+  - name: onIntent
```

Comment:
Is the event called `onIntent`? This will cause problems with Alloy, since it will be `onOnintent=""` there. Can we consider a better name here?